src/main/java/uk/gov/justice/laa/crime/cfecrime/cma/enums/CaseType.java

```diff
@@ -4,6 +4,7 @@
  * migrated from TOGDATA.CASE_TYPES table
  */
 public enum CaseType {
+    UNKNOWN,
```

Comment:
we can't really do this. We're changing the definition of something that we're not allowed to change

Reply:
Agreed it might cause friction down the line being out of step with the MAAT db's list of values.

And this is only being introduced to use in tests, so not ideal.